UI: Fix color select buttons with Yami

cg2121 opened the pull request:
### Description
The color select buttons would not be rendered properly with the Yami based themes.

Before:
![Screenshot from 2022-08-09 07-33-15](https://user-images.githubusercontent.com/19962531/183648132-d3a0bb56-9c7f-47f1-bdb6-747c4b1b24df.png)

After:
![Screenshot from 2022-08-09 07-26-50](https://user-images.githubusercontent.com/19962531/183648171-f52bf456-3cc1-4d50-adfc-768b431a49f4.png)

### Motivation and Context
Found bug when using color select

### How Has This Been Tested?
Set source color to make sure the buttons looked good

### Types of changes
- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

RytoEX commented:
I cannot find an entry in the Qt docs for a "toolButton" property for QPushButtons. Do you have any references available? What is it for?

cg2121 commented:
@Warchamp7 added it with https://github.com/obsproject/obs-studio/commit/8a3abf1de78e7c52074155a5c7d94afa72872bdf

Warchamp7 commented:
The goal of this fix is fine, but I'm not sure if the toolButton styling property is the correct way to go about it. Feels like a hack

Can this not be accomplished with UI file changes alone?

cg2121 commented:
Since we need to set padding on the buttons, toolButton was set to true, so we don't have to add to the qss files, since it is already there. The toolButton property works good for buttons without text, like the color buttons are.

cg2121 committed:
UI: Fix color select buttons with Yami

The color select buttons would not be rendered properly with the
Yami based themes.
